src/db/meta.rs

```diff
@@ -50,22 +51,22 @@ pub struct Meta {
     pub recyclebin_enabled: Option<bool>,
 
     /// A UUID for the recycle bin group
-    pub recyclebin_uuid: Option<String>,
+    pub recyclebin_uuid: Option<Uuid>,
```

Comment:
@sseemayer I'm tempted to remove the `_uuid` here as well, but I feel like just `recyclebin` might be confusing :thinking: . Either way, I think we should either add the `_uuid` suffix everywhere or not use it at all. Maybe a compromise would be to use `_id` instead?

Reply:
See the [other comment](https://github.com/sseemayer/keepass-rs/pull/133#discussion_r1103647333). Probably a good idea, but we should then probably rethink the whole public interface and come up with a consistent structure and naming scheme.